control/src/main/scala/com/lynbrookrobotics/potassium/control/PIDF.scala

```diff
@@ -26,40 +28,66 @@ case class PIDFConfig[S <: Quantity[S],
                       U <: Quantity[U]](kp: Ratio[U, S], ki: Ratio[U, I], kd: Ratio[U, D], kf: Ratio[U, S])
 
 object PIDF {
-  def proportionalControl[S <: Quantity[S], U <: Quantity[U]](signal: PeriodicSignal[S],
-                                                              target: PeriodicSignal[S],
-                                                              gain: Signal[Ratio[U, S]]): PeriodicSignal[U] = {
-    signal.zip(target).map(p => (p._2 - p._1) ** gain.get)
+
+  /**
+    * most control loops have current values updating with greater periodicity
+    * than the target values. This method returns an error that updates in the
+    * manner described in AsyncZipped stream, with the periodicity of current
+    * @param current
+    * @param target
+    * @tparam S
+    * @return
+    */
+  def asyncError[S <: Quantity[S]](current: Stream[S], target: Stream[S]): Stream[S] = {
+    throw new NotImplementedException()
+  }
+
+  // TODO: Use zipAsync instead of zip as used in minus
```

Comment:
Use syncTo when  separate target and current periodicity. Add scaladocs to explain